data/actor-inspector/factories.js

```diff
@@ -56,22 +62,31 @@ var FactoryTable = React.createClass({
  * TODO docs
  */
 var FactoryList = React.createClass({
+  getInitialState: function() {
+    return {
+      packet: [
+        {factories: {}},
+        {factories: {}}
```

Comment:
@Honza or better: I'm going to update the branch and open this pull request on the firebug/firebug.next repo